cmsl1t/analyzers/jetMet_analyzer.py

```diff
@@ -361,8 +361,8 @@ def fill_histograms(self, entry, event):
         # pileup = self._lumiMu[(event['run'], event['lumi'])]
         pileup = 51
         # print pileup
-        if pileup >= 60 or pileup < 50:
-            return True
+        # if pileup >= 60 or pileup < 50:
+        #    return True
```

Comment:
This section looks quite experimental or unfinished.  The pileup is hard-coded to be 51 as well.  Seems funny to me.  Is it really ready to be merged in this state?

Reply:
It is going to be merged into `dev` which is a step-by-step effort to enable Python 3.
In https://github.com/kreczko/cms-l1t-analysis/tree/kreczko-all20X-debug I am pushing this further to the actual histogram filling stage.